[2022-03-18 19:48:55,142] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.processing_task.task_3 2022-03-18T14:18:08.915997+00:00 [queued]>
[2022-03-18 19:48:55,231] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.processing_task.task_3 2022-03-18T14:18:08.915997+00:00 [queued]>
[2022-03-18 19:48:55,232] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 19:48:55,232] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2022-03-18 19:48:55,235] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 19:48:55,265] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_3> on 2022-03-18T14:18:08.915997+00:00
[2022-03-18 19:48:55,277] {standard_task_runner.py:52} INFO - Started process 26639 to run task
[2022-03-18 19:48:55,300] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag.processing_task', 'task_3', '2022-03-18T14:18:08.915997+00:00', '--job-id', '26', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpcxtfc0nk', '--error-file', '/tmp/tmp91aiwnmi']
[2022-03-18 19:48:55,304] {standard_task_runner.py:77} INFO - Job 26: Subtask task_3
[2022-03-18 19:48:55,801] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.processing_task.task_3 2022-03-18T14:18:08.915997+00:00 [running]> on host pc
[2022-03-18 19:48:56,097] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag.processing_task
AIRFLOW_CTX_TASK_ID=task_3
AIRFLOW_CTX_EXECUTION_DATE=2022-03-18T14:18:08.915997+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-18T14:18:08.915997+00:00
[2022-03-18 19:48:56,099] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2022-03-18 19:48:56,110] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-03-18 19:48:56,129] {subprocess.py:75} INFO - Output:
[2022-03-18 19:48:59,135] {subprocess.py:83} INFO - Command exited with return code 0
[2022-03-18 19:48:59,213] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=parallel_dag.processing_task, task_id=task_3, execution_date=20220318T141808, start_date=20220318T141855, end_date=20220318T141859
[2022-03-18 19:48:59,395] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-03-18 19:48:59,440] {local_task_job.py:151} INFO - Task exited with return code 0
